packages/vats/src/core/chain-behaviors.js

```diff
@@ -146,9 +147,24 @@ export const bridgeProvisioner = async ({
     async fromBridge(_srcID, obj) {
       switch (obj.type) {
         case 'PLEASE_PROVISION': {
-          const { nickname, address, powerFlags } = obj;
-          return E(provisioning)
-            .pleaseProvision(nickname, address, powerFlags)
+          const { nickname, address, powerFlags: rawPowerFlags } = obj;
+          const powerFlags = rawPowerFlags || [];
+          let provisionP;
+          if (powerFlags.includes(PowerFlags.SMART_WALLET)) {
+            // Only provision a smart wallet.
+            provisionP = E(bridgeManager).fromBridge(
+              BRIDGE_ID.PROVISION_SMART_WALLET,
+              obj,
```

Comment:
Where does this message actually end up? I do not see any bridge handler registered for it since `boot-psm` sets `walletBridgeId` to `BridgeId.PROVISION`.